cirq-google/cirq_google/engine/asyncio_executor.py

```diff
@@ -40,6 +41,13 @@ def __init__(self) -> None:
 
     @staticmethod
     async def _main(loop_future: duet.AwaitableFuture) -> None:
+        def handle_exception(loop, context) -> None:
```

Comment:
Is this function called somewhere?

Reply:
Oops, forgot to actually set this as the exception handler. Fixed.